requirements/pip.txt

```diff
@@ -80,7 +80,6 @@ django-formtools==2.1
 # https://github.com/rtfd/readthedocs.org/issues/3999
 docker==3.1.3  # pyup: ignore
 
-django-textclassifier==1.0
```

Comment:
This is not needed anymore.

Reply:
I believe this is used for spam project detection. I do worry that we'd get more spam if we don't run any detection. Perhaps we could run it on the name of the project. @agjohnson?

Reply:
@davidfischer 
There's an open PR for this. #5038